documentation/modules/running-migration-plan.adoc

```diff
@@ -21,19 +21,40 @@ The *Plans* list displays the source and target providers, the number of virtual
 . Click *Start* beside a migration plan to start the migration.
 . Click *Start* in the confirmation window that opens.
 +
-The *Migration details by VM* screen opens, displaying the migration's progress
+The plan's *Status* changes to *Running* and migration's progress is displayed.
 +
 Warm migration only:
 
 * The precopy stage starts.
 * Click *Cutover* to complete the migration.
 
-. If the migration fails:
-.. Click *Get logs* to retrieve the migration logs.
-.. Click *Get logs* in the confirmation window that opens.
-.. Wait until *Get logs*  changes to *Download logs* and then click the button to download the logs.
+. Click the links in the migration's *Status* to see its overall status and the status of each VM:
 
-. Click a migration's *Status*, whether it failed or succeeded or is still ongoing, to view the details of the migration.
+** The link on the left indicates whether the migration failed, succeeded, or is ongoing and reports the number of VMs whose migration succeeded, failed, or was canceled.
```

Comment:
```suggestion
** The link on the left indicates whether the migration failed, succeeded, or is ongoing. The link also reports the number of VMs whose migration succeeded, failed, or was canceled.
```

Reply:
@mpershina Did you see Nunzy's comment on this?